providers/sendinblue/src/lib/sendinblue.provider.ts

```diff
@@ -54,6 +56,75 @@ export class SendinblueEmailProvider implements IEmailProvider {
     };
   }
 
+  getMessageId(body: any | any[]): string[] {
+    if (Array.isArray(body)) {
+      return body.map((item) => item['message-id']);
+    }
+
+    return [body['message-id']];
+  }
+
+  parseEventBody(
+    body: any | any[],
+    identifier: string
+  ): IEmailEventBody | undefined {
+    if (Array.isArray(body)) {
+      body = body.find((item) => item['message-id'] === identifier);
+    }
+
+    if (!body) {
+      return undefined;
+    }
+
+    const status = this.getStatus(body.event);
+
+    if (status === undefined) {
+      return undefined;
+    }
+
+    return {
+      status: status,
+      date: body.date,
```

Comment:
What format do this date have?

Reply:
The date was formatted as `YYYY-MM-DD HH:mm:ss`. Is there are standard format that is used for in this? ISO string perhaps?